website/content/docs/agent/wal-logstore/enable.mdx

```diff
@@ -7,30 +7,7 @@ description: >-
 
 # Enable the experimental WAL LogStore backend
 
-This topic describes how to safely configure and test the WAL backend in your Consul deployment.
-
-The overall process for enabling the WAL LogStore backend for one server consists of the following steps. In production environments, we recommend starting by enabling the backend on a single server . If you eventually choose to expand the test to further servers, you must repeat these steps for each one.
-
-1. Enable log verification.
-1. Select target server to enable WAL.
-1. Stop target server gracefully.
-1. Remove data directory from target server.
-1. Update target server's configuration.
-1. Start the target server.
-1. Monitor target server raft metrics and logs.
-
-!> **Experimental feature:** The WAL LogStore backend is experimental and may contain bugs that could cause data loss. Follow this guide to manage risk during testing.
-
-## Requirements
-
-- Consul v1.15 or later is required for all servers in the datacenter. Refer to the [standard upgrade procedure](/consul/docs/upgrading/instructions/general-process) and the [1.15 upgrade notes](/consul/docs/upgrading/upgrade-specific#consul-1-15-x) for additional information.
-- A Consul cluster with at least three nodes are required to safely test the WAL backend without downtime.
-
-We recommend taking the following additional measures:
-
-- Take a snapshot prior to testing.
-- Monitor Consul server metrics and logs, and set an alert on specific log events that occur when WAL is enabled. Refer to [Monitor Raft metrics and logs for WAL](/consul/docs/agent/wal-logstore/monitoring) for more information.
-- Enable WAL in a pre-production environment and run it for a several days before enabling it in production.
+WAL LogStore is now enabled by default
```

Comment:
We're still ignoring config if there's a BoltDB file found - should we call that out here (new installs only), and keep some instructions to transition existing servers if desired?

Reply:
@boruszak Can you please check the wording in here?